addons/payment_paypal/models/payment.py

```diff
@@ -183,8 +183,10 @@ def _paypal_form_get_invalid_parameters(self, data):
             # different than the business email. Therefore, if you want such a configuration in your Paypal, you are then obliged to fill
             # the Merchant ID in the Paypal payment acquirer in Odoo, so the check is performed on this variable instead of the receiver_email.
             # At least one of the two checks must be done, to avoid fraudsters.
-            if data.get('receiver_email') != self.acquirer_id.paypal_email_account:
+            if data.get('receiver_email') and data.get('receiver_email') != self.acquirer_id.paypal_email_account:
                 invalid_parameters.append(('receiver_email', data.get('receiver_email'), self.acquirer_id.paypal_email_account))
+            if data.get('business') and data.get('business') != self.acquirer_id.paypal_email_account:
```

Comment:
Hello @dbo-odoo 
Here we have 2 cases.
1) without merchant account-
 here we got merchant's email in business parameter because till now, the merchant didn't receive money in his account, so it can't be receivere_email

2) with merchant account-
 when the merchant has an account, he will directly receive money in his account.
  so here we got receiver  email in response

Reply:
yes indeed